diff-test/core/__tests__/gen.test.ts

```diff
@@ -0,0 +1,8 @@
+import { gen } from '../src/main.js';
```

Comment:
What's this?

Reply:
It's useful to have a test to check model coverage. I've added a comment now.